spec/ScoreSpec.js

```diff
@@ -0,0 +1,108 @@
+describe("Game", function() {
+  var score;
+  var frame;
+  var display;
+
+  beforeEach(function() {
+    game = new Game();
+    frame = new Frame();
```

Comment:
If you're testing Game class, then I would expect frame, display and other dependencies to be mock objects